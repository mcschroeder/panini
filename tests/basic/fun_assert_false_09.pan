assert : { b:𝔹 | b = true } → 𝟙

f20 : unit -> {v:unit|false}
f20 = \x:unit. assert false

f21 : bool -> {v:unit|false}
f21 = \x:bool. assert false

f22 : int -> {v:unit|false}
f22 = \x:int. assert false

f23 : string -> {v:unit|false}
f23 = \x:string. assert false
