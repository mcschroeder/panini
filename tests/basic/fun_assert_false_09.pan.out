tests/basic/fun_assert_false_09.pan:4:1-4: error: unable to verify type of f20
tests/basic/fun_assert_false_09.pan:7:1-4: error: unable to verify type of f21
tests/basic/fun_assert_false_09.pan:10:1-4: error: unable to verify type of f22
tests/basic/fun_assert_false_09.pan:13:1-4: error: unable to verify type of f23